how-to/how-to-delete-local-cache.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How to Delete The Extension for Visual Studio and The Classic Windows Desktop Client Local Cache and Cookies
+description: Step by step tutorial for deleting the Telerik Platform local cache and cookies.
+type: how-to
+page_title: How to Clear AppBuilder Windows Clients Local Cache and Cookies
+slug: delete-local-cache
+position:
+tags: other
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>{{site.ab-s}} ({{site.ab}}) clients</td>
+    <td>Extension for Visual Studio, classic Windows Desktop</td>	
+  </tr>
+</table>
+
+## Description
+
+This article provides the general steps for deleting the Extension for Visual Studio and the classic Windows Desktop client local cache and cookies.
```

Comment:
For better readability, I suggest changing the sentence order here, too. When I am reading it, I feel like I am going to delete the extension before I reach the end of the sentence :)